<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Contactbook</class>
 <widget class="QWidget" name="Contactbook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>é€šè¨ŠéŒ„ç®¡ç†ç³»çµ± - ContactBook</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #f5f5f5;
    font-family: "Microsoft YaHei", "å¾®è»Ÿæ­£é»‘é«”", sans-serif;
    font-size: 10pt;
}

QTableWidget {
    background-color: white;
    alternate-background-color: #f0f8ff;
    selection-background-color: #4a90e2;
    selection-color: white;
    gridline-color: #d0d0d0;
    border: 1px solid #c0c0c0;
    border-radius: 5px;
}

QTableWidget::item {
    padding: 5px;
}

QTableWidget QHeaderView::section {
    background-color: #2c3e50;
    color: white;
    padding: 8px;
    border: none;
    font-weight: bold;
}

QGroupBox {
    background-color: white;
    border: 2px solid #4a90e2;
    border-radius: 8px;
    margin-top: 10px;
    padding-top: 15px;
    font-weight: bold;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 5px 10px;
    background-color: #4a90e2;
    color: white;
    border-radius: 5px;
    left: 10px;
}

QLabel {
    color: #2c3e50;
    font-weight: bold;
}

QLineEdit {
    padding: 8px;
    border: 2px solid #d0d0d0;
    border-radius: 5px;
    background-color: white;
    selection-background-color: #4a90e2;
}

QLineEdit:focus {
    border: 2px solid #4a90e2;
}

QPushButton {
    background-color: #4a90e2;
    color: white;
    border: none;
    border-radius: 5px;
    padding: 10px 20px;
    font-weight: bold;
    min-width: 80px;
}

QPushButton:hover {
    background-color: #357abd;
}

QPushButton:pressed {
    background-color: #2c5f8d;
}

QPushButton#btnAdd {
    background-color: #27ae60;
}

QPushButton#btnAdd:hover {
    background-color: #229954;
}

QPushButton#btnEdit {
    background-color: #f39c12;
}

QPushButton#btnEdit:hover {
    background-color: #d68910;
}

QPushButton#btnDelete {
    background-color: #e74c3c;
}

QPushButton#btnDelete:hover {
    background-color: #c0392b;
}

QPushButton#btnExit {
    background-color: #7f8c8d;
}

QPushButton#btnExit:hover {
    background-color: #5f6c6d;
}

QPushButton#btnSearch {
    background-color: #8e44ad;
}

QPushButton#btnSearch:hover {
    background-color: #7d3c98;
}</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QGroupBox" name="searchGroupBox">
     <property name="title">
      <string> æœå°‹åŠŸèƒ½ </string>
     </property>
     <layout class="QHBoxLayout" name="searchLayout">
      <item>
       <widget class="QLabel" name="lblSearch">
        <property name="text">
         <string>æœå°‹:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtSearch">
        <property name="placeholderText">
         <string>è¼¸å…¥å­¸è™Ÿã€å§“åæˆ–é›»è©±é€²è¡Œæœå°‹...</string>
        </property>
        <property name="toolTip">
         <string>è¼¸å…¥é—œéµå­—æœå°‹è¯çµ¡äººè³‡è¨Š</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSearch">
        <property name="text">
         <string>ğŸ” æœå°‹</string>
        </property>
        <property name="toolTip">
         <string>æœå°‹è¯çµ¡äºº</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClearSearch">
        <property name="text">
         <string>æ¸…é™¤</string>
        </property>
        <property name="toolTip">
         <string>æ¸…é™¤æœå°‹æ¢ä»¶ï¼Œé¡¯ç¤ºæ‰€æœ‰è¯çµ¡äºº</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="toolTip">
      <string>è¯çµ¡äººåˆ—è¡¨ - é»æ“Šé¸æ“‡è¦ç·¨è¼¯æˆ–åˆªé™¤çš„è¯çµ¡äºº</string>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="inputGroupBox">
     <property name="title">
      <string> è¯çµ¡äººè³‡è¨Š </string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>15</number>
      </property>
      <property name="verticalSpacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="topMargin">
       <number>15</number>
      </property>
      <property name="rightMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>15</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lblStudentId">
        <property name="text">
         <string>å­¸è™Ÿ:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtStudentId">
        <property name="placeholderText">
         <string>è«‹è¼¸å…¥å­¸è™Ÿ</string>
        </property>
        <property name="toolTip">
         <string>å­¸ç”Ÿå­¸è™Ÿ</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblClass">
        <property name="text">
         <string>ç­ç´š:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtClass">
        <property name="placeholderText">
         <string>è«‹è¼¸å…¥ç­ç´š</string>
        </property>
        <property name="toolTip">
         <string>å­¸ç”Ÿç­ç´š</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblName">
        <property name="text">
         <string>å§“å:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="txtName">
        <property name="placeholderText">
         <string>è«‹è¼¸å…¥å§“å</string>
        </property>
        <property name="toolTip">
         <string>å­¸ç”Ÿå§“å</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblPhone">
        <property name="text">
         <string>é›»è©±:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="txtPhone">
        <property name="placeholderText">
         <string>è«‹è¼¸å…¥é›»è©±è™Ÿç¢¼</string>
        </property>
        <property name="toolTip">
         <string>è¯çµ¡é›»è©±</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <spacer name="leftSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="text">
        <string>â• æ–°å¢</string>
       </property>
       <property name="toolTip">
        <string>æ–°å¢è¯çµ¡äººè³‡è¨Š</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEdit">
       <property name="text">
        <string>âœï¸ ç·¨è¼¯</string>
       </property>
       <property name="toolTip">
        <string>ç·¨è¼¯é¸ä¸­çš„è¯çµ¡äºº</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDelete">
       <property name="text">
        <string>ğŸ—‘ï¸ åˆªé™¤</string>
       </property>
       <property name="toolTip">
        <string>åˆªé™¤é¸ä¸­çš„è¯çµ¡äºº</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExport">
       <property name="text">
        <string>ğŸ’¾ åŒ¯å‡º</string>
       </property>
       <property name="toolTip">
        <string>åŒ¯å‡ºè¯çµ¡äººè³‡è¨Šåˆ°æª”æ¡ˆ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnImport">
       <property name="text">
        <string>ğŸ“‚ åŒ¯å…¥</string>
       </property>
       <property name="toolTip">
        <string>å¾æª”æ¡ˆåŒ¯å…¥è¯çµ¡äººè³‡è¨Š</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExit">
       <property name="text">
        <string>âŒ çµæŸ</string>
       </property>
       <property name="toolTip">
        <string>é—œé–‰ç¨‹å¼</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="rightSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
